cli/provider/cmd.go

```diff
@@ -173,6 +173,27 @@ func (c *CmdConfigurator) AddFlags(cmd *cobra.Command) error {
 	var err error
 	cmd.Flags().SetNormalizeFunc(aliasNormalizeFunc)
 	switch cmd.Name() {
+	case "generate", "download":
+		cmd.Flags().StringSliceP("services", "s", c.cfg.Contract.Services, "Specify the services for which to generate/download contracts")
+		cmd.Flags().StringSliceP("tests", "t", c.cfg.Contract.Tests, "Specify the tests for which to generate/download contracts")
+
+		cmd.Flags().StringP("path", "p", c.cfg.Contract.Path, "Specify the path to generate/download contracts")
+		err := cmd.MarkFlagRequired("path")
```

Comment:
Is this even required, are we not taking default values, for eg: if "generate", just generate the schema for tests and mocks under the schema folder? And if the "download" command then you can store the (driven=="client") mocks and mocks schema under the service name in the same root directory.